ledger/src/blockstore_meta.rs

```diff
@@ -331,11 +345,12 @@ impl ErasureMeta {
                     num_coding: usize::from(shred.num_coding_shreds().ok()?),
                 };
                 let first_coding_index = u64::from(shred.first_coding_index()?);
-                let erasure_meta = ErasureMeta {
+                let merkle_root = Hash::default();
```

Comment:
next change in master will populate this from the coding shred, but I don't think that needs to be part of this backport change.